MISRA.md

```diff
@@ -128,10 +143,28 @@ _Ref 17.2.1_
         have a secondary child socket thereby limiting the number of recursive
         calls to one.
 
+#### Rule 20.5
+_Ref 20.5.1_
+
+- MISRA C-2012 Rule 20.5 warns against the use of #undef
+        However, in this case, it must be used to make sure some macros are
+        defined to a certain value
+
 #### Rule 20.10
 _Ref 20.10.1_
 
 - MISRA C-2012 Rule 20.10 warns against the use of ## concatination operator.
         However, in this case, it must be used to support compile time
         assertions in case the preprocessor does not suppport sizeof. This
         operation (assert) has no runtime execution.
+
+#### Rule 21.6
+_Ref 21.6.1_
+
+- MISRA C-2012 Rule 21.6 warns about the use of standard library input/output
```

Comment:
This justification should mention how we prevent misuse of printf-style functions. Mentioning "-Wformat-security" and/or specific Wformat options that are used when building unit tests would be helpful here.